src/components/header/index.tsx

```diff
@@ -2,21 +2,36 @@ import { Alignment, Button, Icon, Navbar as BPNavbar } from "@blueprintjs/core";
 import * as React from "react";
 import { Link } from "react-router-dom";
 import HeaderPatientName from "components/header/headerPatientName";
+import { DATA_TEST } from "constants/dataTest";
 
 import "./style.css";
 
-import { paths } from "constants/routes";
+import { ROUTE_HOME } from "constants/routes";
 
 interface Props {
   patientId?: string;
 }
 
+interface Patient {
+  firstName: string;
+  lastName: string;
+  id: string;
+  age: number;
+  medicalHistory: string;
+  allergies: string;
+}
+
 const Header = ({ patientId }: Props) => {
+  var patient;
+
+  for (var i in DATA_TEST)
+    if (DATA_TEST[i].id === patientId) patient = DATA_TEST[i];
```

Comment:
`for` loops are not very common in js, becaus you have a lot oh helper functions on arrays/objects.
Instead, you may use: `const patient = DATA_TEST.find(p => p.id === patientId)`